### DISCUSSION TITLE
Duplicate already created items

### DISCUSSION THREAD

**Discussion by laawn** [chhoumann/quickadd#205] [Ideas]
There needs to be a simple way to duplicate already created items. There are plenty of times I've already created a macro that I want to reuse but change a small portion (i.e. the output file). Currently, AFAIK, I have to recreate the entire thing. Would be great if I could simply duplicate with a click.

Thanks for making such an amazing plugin!

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that.

This shipped: every choice has a one-click Duplicate (also available by right-clicking the choice in Settings -> QuickAdd). The copy gets a "(copy)" suffix and fresh internal ids, and duplicating a Multi copies everything nested inside it - so "same macro, different output file" is now duplicate, rename, tweak. One-click duplication landed in v0.10.0, and the right-click menu in 2.10.0.

Thanks for the kind words back then. Closing as resolved.
